### PR TITLE
Bumps UI to 2.27.8

### PR DESCRIPTION
**Pull request by juliangieseke**

### CODE REVIEW

File: stable/kommander/requirements.lock [hunk 1]
```diff
@@ -4,10 +4,10 @@ dependencies:
   version: 0.1.4
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: v0.4.6
```
Comment (GeorgiSTodorov) [inline stable/kommander/requirements.lock]: Was the "v" a mistake or are we changing the format?

Comment (juliangieseke) [reply stable/kommander/requirements.lock]: both works, I think its depedning on different helm versions? its a lockfile - wasnt a manual change.

**Comment by juliangieseke** [comment]
@shaneutt all thats left is your review 😇

**Comment by juliangieseke** [comment]
we're working on getting UI out of this repo, best automation is `noop` :D

**Pull request merged**